proto/relay.proto

```diff
@@ -9,17 +9,35 @@ import "self_serve/orb.proto";
 
 // Service Definitions
 service OrbService {
-  rpc OrbConnect(stream OrbEvent) returns (stream OrbCommand);
+  rpc OrbConnect(stream RelayMessage) returns (stream RelayMessage);
 }
 
 service AppService {
-  rpc AppConnect(stream AppEvent) returns (stream AppCommand);
+  rpc AppConnect(stream RelayMessage) returns (stream RelayMessage);
 }
 
 service BackendService {
   rpc RequestOrbUpdateConfig(config.OrbUpdateConfigRequest) returns (config.OrbUpdateConfigResponse);
 }
 
+message RelayPayload {
+  oneof payload {
+    OrbConnectRequestEvent orb_connect_request_event = 1;
+    AppConnectRequestEvent app_connect_request_event = 2;
+    OrbCommand orb_command = 3;
+    OrbEvent orb_event = 4;
+    AppCommand app_command = 5;
+    AppEvent app_event = 6;
+    BackendEvent backend_event = 7;
+    BackendCommand backend_command = 8;
+  }
+}
+
+message RelayMessage {
```

Comment:
would it make sense to have an optional origin string too?

Reply:
Definitely could make sense. I'll hold off on adding until we see a viable use case for it. Easy to add :)

Reply:
Actually... Don't we need it even now? Imagine self serve flow scenario. `App` sends message to `Orb`. `Orb` needs to know where to respond. 

Our self serve integration tests rely on having both clients know each other `app_id` and `orb_id` - that's just how we wrote tests. 

In real life scenario, `app` or `orb` that receive message, will not know the `sender` - we should add it now.